featurehub-javascript-client-sdk/app/feature_state_holders.ts

```diff
@@ -4,10 +4,10 @@ import { ClientContext } from './client_context';
 import { InternalFeatureRepository } from './internal_feature_repository';
 import { ListenerUtils } from './listener_utils';
 
-export class FeatureStateBaseHolder implements FeatureStateHolder {
+export class FeatureStateBaseHolder<T = any>  implements FeatureStateHolder<T> {
   protected internalFeatureState: FeatureState | undefined;
   protected _key: string;
-  protected listeners: Map<Number, FeatureListener> = new Map<Number, FeatureListener>();
+  protected listeners: Map<number, FeatureListener> = new Map<number, FeatureListener>();
```

Comment:
If you are adding tests, you may wish to change the ClientFeatureRepository to store the same Map<number instead of Map<Number so we are consistent - its a good change, its just about consistency.